config.json

```diff
@@ -178,6 +178,18 @@
       ]
     },
     {
+      "uuid": "53893984-080f-a180-5fa0-1af0fdb5ae3b6e23169",
```

Comment:
the last portion is too long (should be 12 hexadecimal digits but has 19 by my count), so this is not a valid https://en.wikipedia.org/wiki/Universally_unique_identifier (I am assuming this is what a UUID is)

Reply:
Sorry. I generated this with [configlet](https://github.com/exercism/configlet/), but I hadn't seen that [it might not generate valid UUIDs](https://github.com/exercism/configlet/issues/99). This should be fixed now.